catalogue/webapp/test/e2e/urls.ts

```diff
@@ -0,0 +1,14 @@
+export const baseUrl = (): string => {
+  switch (process.env.NODE_ENV) {
+    case 'production':
+      return 'https://wellcomecollection.org';
+    case 'staging':
+      return 'https://www-stage.wellcomecollection.org';
+    default:
+      return `http://localhost:${process.env.PORT || 3000}`;
```

Comment:
We can just hardcode this to 3000, we never configure it differently.

Reply:
nice!